Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -9,30 +9,28 @@ public struct PaymentFormFactory {
   internal static var sessionCorrelationID = ""
 
   public static func buildViewController(configuration: PaymentFormConfiguration,
-                                         style: PaymentStyle) -> UIViewController {
+                                         style: PaymentStyle,
+                                         cardTokenResultClousure: @escaping (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void) -> UIViewController {
```

Comment:
`cardTokenResultClousure` is the naming convention right ? calling a closure ending with Closure keyword. Can you please guide me with some examples I can refer.

Reply:
Following apple naming convention, it's changed to `completionHandler`